ui/src/App.vue

```diff
@@ -53,6 +55,10 @@ html {
   margin-left: v-bind(contentMargin);
   transition: margin-left 0.28s ease-in-out v-bind(ease);
   max-width: calc(100% - v-bind(maxWidth));
+
+  table {
+    width: 100%;
+  }
```

Comment:
Looks like this can be removed on a bunch of tables now: DCBTable, EventsTable, IPInterfaceTable, NodesTable, OutagesTable, SNMPInterfacesTable, GraphDataTable